### PR TITLE
Bundle of performance improvement

### PR DESCRIPTION
**Pull request by FranckKe**
Configure front bundle size
Add gzip compression for assets in nginx
Update front packages
Update devise to fix cve

### CODE REVIEW

File: dockerfiles/nginx.conf [hunk 1]
```diff
@@ -13,10 +13,41 @@ server {
   root /usr/src/app/public;
   index index.html;
 
+  # gzip
+  gzip on;
+  gzip_vary on;
+  gzip_proxied any;
+  gzip_comp_level 6;
+  gzip_types text/plain text/css text/xml application/json application/javascript application/rss+xml application/atom+xml image/svg+xml;
```
Comment (mdouchement) [inline dockerfiles/nginx.conf]: Wondering if basic hhtp library handles gzipped `application/json` data

Comment (FranckKe) [reply dockerfiles/nginx.conf]: `gunzip: on` should take care of this. Adding it.
https://docs.nginx.com/nginx/admin-guide/web-server/compression/#enabling-decompression

**Comment by FranckKe** [comment]
@mdouchement Should compression be in nginx or traeffik ?

File: dockerfiles/nginx.conf (hunk 1 above)
Comment (mdouchement) [reply dockerfiles/nginx.conf]: Did you try this config with Python's `requests` for example?

Comment (FranckKe) [reply dockerfiles/nginx.conf]: Yes, it works as expected. The config is up on alpha if you want to check.

**Comment by mdouchement** [comment]
@FranckKe nginx is fine, we can apply dedicated rules for mersea.

**Pull request merged**